### PR TITLE
producer: add flag to republish buried jobs in mentions iter

### PR DESCRIPTION
**Pull request by erizocosmico**

### CODE REVIEW

File: cli/borges/producer.go [hunk 1]
```diff
@@ -56,6 +57,12 @@ func (c *producerCmd) jobIter(b queue.Broker) (borges.JobIter, error) {
 		if err != nil {
 			return nil, err
 		}
+
+		if c.RepublishBuried {
+			if err := q.RepublishBuried(); err != nil {
```
Comment (ajnavarro) [inline cli/borges/producer.go]: I think this will fail if the queue is not already created.

**Comment by bzz** [comment]
Looks great to me.

@erizocosmico do you think it might be worth adding a new sub-command to the README ?

File: cli/borges/producer.go (hunk 1 above)
Comment (erizocosmico) [reply cli/borges/producer.go]: Anything we can do about that? Anyway, it's a flag you have to set, it does not make sense to set it if you know there is nothing in the buried queue.

File: repos.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+git://127.0.0.1/
```
Comment (ajnavarro) [inline repos.txt]: leftover?

**Pull request merged**